[BUG] Fix make install command

**Describe the bug**
<!--
A clear and concise description of what the bug is.
-->
`setup.py` file does not exists

https://github.com/sktime/sktime/blob/5f0ab729c6d08ec22884f88e1be4f0145f7e56d5/Makefile#L22-L25
**To Reproduce**
<!--
Add a Minimal, Complete, and Verifiable example (for more details, see e.g. https://stackoverflow.com/help/mcve

If the code is too long, feel free to put it in a public gist and link it in the issue: https://gist.github.com
-->
Try doing make install. It gives:
```
(venv) f@f:~/w/oss/e/sktime$ make install
python3 setup.py build_ext --inplace && python setup.py install --user
python3: can't open file '/home/f/w/oss/e/sktime/setup.py': [Errno 2] No such file or directory
make: *** [Makefile:24: install] Error 2
(venv) f@f:~/w/oss/e/sktime$ 


```

**Expected behavior**

Correct installation should happen.


good first issues :)

--- a/Makefile
+++ b/Makefile
@@ -20,8 +20,8 @@ help:
 release: ## Make a release
 	python3 $(BUILD_TOOLS)/make_release.py
 
-install: ## Install for the current user using the default python command
-	python3 setup.py build_ext --inplace && python setup.py install --user
+install: ## Install editable into the active Python environment
+	python3 -m pip install -e .
 
 test: ## Run unit tests
 	-rm -rf ${TEST_DIR}

